src/makeCalculator.js

```diff
@@ -4,7 +4,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+  const calculator = {
+    operate(callback, number) {
+      if (typeof callback === 'function') {
+        result = callback(result, number);
+      }
+
+      return this;
+    },
+
+    getResult() {
+      return result;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
```

Comment:
The `multiply` method returns the product of `a` and `b` without affecting the internal `result`. If the goal is to update the `result`, modify the `result` variable accordingly.